sktime/forecasting/base/_base.py

```diff
@@ -137,6 +137,8 @@ def fit(self, y, X=None, fh=None):
         }
 
         y = check_series(y, **check_y_args, var_name="y")
+
+        self.y_mtype = mtype(y)
```

Comment:
let's make this a private variable
```suggestion
        self._y_mtype = mtype(y)
```

Reply:
I thought the converter store could handle this?

Reply:
no, currently it only stores information from lossy conversions such as data frame column names - not the original mtype.
It might be an idea to change the behaviour that it even remembers the original mtype, but what would the syntax be?

Reply:
Anyway, *this* is a prime case for a private variable.

I've changed it to private, and to be clearer on the content, to `self._y_mtype_last_seen`